static/js/apps/explore/app.tsx

```diff
@@ -197,11 +197,16 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       }
       if (
         shouldSetQuery &&
-        pageMetadata.mainTopic?.name &&
+        !_.isEmpty(pageMetadata.mainTopics) &&
         pageMetadata.place.name
       ) {
-        const q = `${pageMetadata.mainTopic.name} in ${pageMetadata.place.name}`;
-        setQuery(q);
+        if (pageMetadata.mainTopics.length == 2) {
+          const q = `${pageMetadata.mainTopics[0].name} vs. ${pageMetadata.mainTopics[1].name} in ${pageMetadata.place.name}`;
```

Comment:
will topics always have names? In the old code, we checked for a name before actually setting the query so not sure if that was because sometimes topics might not have a name?

Reply:
sg, I can make that change in follow up...